### ISSUE TITLE
how to add another org through sdk?

### ISSUE THREAD

**Issue by dexhunter** [hyperledger/fabric-sdk-py#78]
I got some requests on how to add another organization through sdk, I will check the issue and update the tutorial.

**Comment by dexhunter** [comment]
A reference: https://hyperledger-fabric.readthedocs.io/en/release-2.0/channel_update_tutorial.html

**Comment by 00Kai0** [comment]
Does the python have this function? or need help to add this function?

**Comment by dexhunter** [comment]
@00Kai0 I think all the parts are ready but there is no wrap-up function

**Comment by 00Kai0** [comment]
yeah, need a tutorial about it. Are you working on it?

**Comment by dexhunter** [comment]
Not currently, I'm still writing my thesis. If depending on me, it will be June or July. So if anyone is available, feel free to pick this task up.

**Comment by 00Kai0** [comment]
Yeah, I want to do if I can. But I don't find any task list in jira.
Is it because the python sdk is too new? Is it still in preview?

**Comment by dexhunter** [comment]
we migrate from jira to github issues now.

**Comment by dexhunter** [comment]
But you can create an issue on jira as well if you want to. [Link here](https://jira.hyperledger.org/projects/FABP/issues)

**Comment by 00Kai0** [comment]
that's great, thank you

**Comment by tobigiwa** [comment]
@dexhunter is this still open?
